### PR TITLE
LIME-523: Add images based on deploy

### LINKED ISSUE #523: Enhancement: Change images into screenshots of the real app and update resources in footer on the Landing page
- [ ] Update images of the Overview page and Sign in page with screenshots of the Lime app
- [ ] Update Resorses in Footer. Resourses should inform user that Lime app is Powered by Strava, Powered by Google Fit, Powered by AI. Consider solution like below

Image

Image

### PR DESCRIPTION
**Pull request by DavidCharif**
image
image
image
image

### CODE REVIEW

**Comment by DavidCharif** [comment]
Resources update
image

File: frontend/src/bundles/landing/pages/landing.tsx [hunk 1]
```diff
@@ -406,9 +427,15 @@ const Landing = (): JSX.Element => {
...
+                                </li>
+                                <li>
+                                    <u>Powered</u> by <b>Google fit</b>
+                                </li>
+                                <li>
+                                    <u>Powered</u> by <b>AI</b>
+                                </li>
                                 <Link to={AppRoute.ROOT}>
                                     See all resources
                                 </Link>
```
Comment (nikita-remeslov) [inline frontend/src/bundles/landing/pages/landing.tsx]: we dont need it

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

File: frontend/src/bundles/landing/pages/landing.tsx [hunk 2]
```diff
@@ -406,9 +427,15 @@ const Landing = (): JSX.Element => {
                                 Resources
```
Comment (nikita-remeslov) [inline frontend/src/bundles/landing/pages/landing.tsx]: this is not Resources

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Comment by DavidCharif** [comment]
image
image
image

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**